lrauv_ignition_plugins/test/helper/LrauvTestFixture.hh

```diff
@@ -171,10 +171,32 @@ class LrauvTestFixture : public ::testing::Test
     return pid;
   }
 
+  public: static void ExportLogs(std::string _target)
+  {
+    int res = system("sudo /home/developer/lrauv_ws/src/lrauv/lrauv_ignition_plugins/plots/unserialize_for_plotting.sh");
```

Comment:
Is sudo needed?

Reply:
I'm still iterating on this to figure out the docker permissions.